Cargo.toml

```diff
@@ -17,13 +17,23 @@ log = "0.4"
 clap = "2.33.3"
 regex = "1"
 procfs = "0.8.1"
-actix-web = "3"
-riemann_client = "0.9.0"
+riemann_client = { version = "0.9.0", optional = true }
 hostname = "0.3.1"
 protobuf = "2.20.0"
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-warp10 = "1.0.0"
+serde = { version = "1.0", features = ["derive"], optional = true }
+serde_json = { version = "1.0", optional = true }
+warp10 = { version = "1.0.0", optional = true }
```

Comment:
Maybe you should bump to 1.2.0

Reply:
We sure should. I didn't check the potential changes to make to move to warp10 1.2
Maybe it should be in another PR as this one is already pretty big ?